src/lang/queryAst.ts

```diff
@@ -1058,3 +1058,89 @@ export const valueOrVariable = (variable: KclCommandValue) => {
     ? variable.variableIdentifierAst
     : variable.valueAst
 }
+
+export function findImportNodeAndAlias(
+  ast: Node<Program>,
+  pathToNode: PathToNode
+) {
+  const importNode = getNodeFromPath<ImportStatement>(ast, pathToNode, [
+    'ImportStatement',
+  ])
+  if (
+    !err(importNode) &&
+    importNode.node.type === 'ImportStatement' &&
+    importNode.node.selector.type === 'None' &&
+    importNode.node.selector.alias &&
+    importNode.node.selector.alias?.type === 'Identifier'
```

Comment:
If someone does `import 'foo.kcl'`, there's no alias. So will this code work?

Reply:
I was going to write in that for the cases hanled here we're good, but there's definitely one case that comes to mind that wouldn't be covered here:
1. you type in the import statement, no alias
2. you go with point and click to set transforms on it

Pretty certain that would fail. Only way to achieve this would be to codemod the alias part to start with I think?

Reply:
Do you want to make an issue for this so that we don't forget?

Reply:
Yes, can totally create one for this and a second one for the deletion issue

Reply:
https://github.com/KittyCAD/modeling-app/issues/6599